core/src/main/java/com/homeaway/streamplatform/streamregistry/db/dao/impl/StreamDaoImpl.java

```diff
@@ -126,61 +127,56 @@ public void upsertStream(Stream stream) {
             log.error("Stream '{}' is not valid", stream.getName());
         }
 
```

Comment:
no changes here except, the try catch is removed. SchemaManagerException, InvalidStreamException, StreamCreationException are not handled here, rather thown back to Resource class

Reply:
Yeah. I would like to schedule a pair-programming with you and @vinayakponangi and redo this with pattern moving forward. Please create an issue to cover this.